test/language/templates/tagged/template-object.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.3.7
+description: >
+    The first argument to a tagged template should be a template object as
+    defined by the GetTemplateObject abstract operation.
+includes: [propertyHelper.js]
+---*/
+var templateObject;
+
+(function(parameter) {
+  templateObject = parameter;
+})`${1}`;
+
+templateObject.test262Prop = true;
```

Comment:
Good point. To limit duplication, I've removed just the assertions for "frozen-ness" and created two new files for them: one runs only in strict mode and expects an error, the other runs only in non-strict mode and simply asserts that the assignment is not honored.